my-app/components/navbar/navbar.module.scss

```diff
@@ -0,0 +1,34 @@
+@mixin flex-center($justify:center){
```

Comment:
I think that mixin should have a default value as such that nullify the effect of justify content rather than center because of its usage it seems that you need center in only one place and you are providing the value in the rest of the usage. Improve you mixin logic.